mopidy/core/tracklist.py

```diff
@@ -10,6 +11,12 @@
 logger = logging.getLogger(__name__)
 
 
+class AddAtPosition(enum.Enum):
+    START = enum.auto()
+    AFTER_CURRENT = enum.auto()
+    END = enum.auto()
```

Comment:
This could be a string Enum:

```suggestion
class AddAtPosition(str, enum.Enum):
    START = "START"
    AFTER_CURRENT = "AFTER_CURRENT"
    END = "END"
```

Then you could accept strings to the `at_position` argument and check if they're valid with simply `at_position in AddAtPosition`, and the strings could be converted to enum values with just `AddAtPosition(at_position)`.

Reply:
I deliberately didn't want to use a string Enum because I feel like you lose the benefits of using an Enum from a type checking point of view. I've seen them used before and they've always felt like a kludge.